content/tutorials/coordination.md

```diff
@@ -0,0 +1,122 @@
+
+## Install
+Clone repo and install the package: 
+{{< highlight bash >}}
+$ cd ~/catkin_ws/src/
+$ git clone https://github.com/prl-mushr/mushr_coordination.git
+$ cd mushr_coordination
+$ git clone https://github.com/whoenig/libMultiRobotPlanning.git
+$ cd ~/catkin_ws/
+$ catkin_make
+{{< / highlight >}}
+
+## API
+For adjusting params see `launch/mushr_coordination.launch` it has the environment params for the planner. Change the number of waypoints for each goal by setting parameter `num_waypoint`. Change the car team setting by editing or creating new config file. 
+
+## Running the coordination planner
+
+Run the planner that will set up the environment publishers and subscribers.
+{{< highlight bash >}}
+$ roslaunch mushr_coordination mushr_coordination.launch cars_file:={car's team config file}
+{{< / highlight >}}
+
+For simulation, run `rviz` then and subscribe to planner's topics. Download the [rviz config file](/tutorials/coordination_planner/nhttc.rviz) for an out-of-box rviz setup. This setup subscribes to all the important rostopics and organizes them in categories. 
+{{< highlight bash >}}
+$ rviz
+{{< / highlight >}}
+
+Below is the format for setting the initial position for a team of cars and a set of tasks to complete. For out-of-box examples, look at the **Example of Initializing the Planner** section which utilizes files already part of the repository.
+{{< highlight bash >}}
+//format for launching
+$ roslaunch mushr_coordination init_planner.launch cars_file:={car's team config file} task_file:={task benchmark file}
+{{< / highlight >}}
+
+### Configuration File Example for Car Information
+This `4cars1.yaml` file shows how to edit and create a config file. The file will have the number of agents (`num_agent`) following with `car{n}`'s `name` and `color` in hex. 
+
+Note that, It would be advised to use the same config file for both `mushr_coordination` and `init_planner`.
+```
+num_agent: 4
+car1:
+    name: "car1"
+    color: "FF0000" #red
+car2:
+    name: "car2"
+    color: "00FF00" #green
+car3:
+    name: "car3"
+    color: "0000FF" #blue
+car4:
+    name: "car4"
+    color: "DB921D" #orange
+```
+
+## Example of Initializing the Planner
+
+### Example #1
+In this environment, there are 4 cars and have been given 4 tasks to complete within a 5 by 3 space. 
+{{< highlight bash >}}
+$ roslaunch mushr_coordination mushr_coordination.launch cars_file:=4cars.yaml
+$ rviz
+$ roslaunch mushr_coordination init_planner.launch cars_file:=4cars.yaml task_file:=4cars4tasks5x3.yaml
+{{< / highlight >}}
+
+Each car get assigned to their respective task, in which their accumulated distanse would be minimal.
+
+<img src="/tutorials/coordination_planner/4cars4task5x3.png" width="800" >
```

Comment:
Using the `nhttc.rviz` config file linked above shows the same task assignment and plan but rotated 180 degrees (this happens for every example). Images should be updated to match this.

For reference, here's what I see for this example:
![4tasks5x3](https://user-images.githubusercontent.com/14863795/125573501-578473b6-fe85-478c-9dc9-b0482154dbb0.PNG)